src/torchrunx/agent.py

```diff
@@ -33,7 +37,8 @@ class WorkerArgs:
     local_rank: int
     local_world_size: int
     world_size: int
-    log_file: os.PathLike
+    log_name: str
+    log_host: str
```

Comment:
Can we just set `log_host` to `master_hostname`?

Reply:
Actually `master_hostname` and `log_host` are not the same - `master_hostname` is referring to the 'master' worker, meanwhile `log_host` is the hostname of the launcher.